core/sql/exp/exp_eval.cpp

```diff
@@ -3003,7 +3014,7 @@ ex_expr::exp_return_type ex_expr::evalPCode(PCodeBinary* pCode32,
             if (diagsArea != atp1->getDiagsArea())
                    atp1->setDiagsArea(diagsArea);
 	    if (er == ex_expr::EXPR_ERROR) 
-	      return ex_expr::EXPR_ERROR;
+		return ex_expr::EXPR_ERROR;
```

Comment:
indentation has changed at many places with the new code. 
Not sure what editor was used.
Will be better to keep the older indentation.